### PR TITLE
Typo sins

### PR DESCRIPTION
**Pull request by klokane**
Addresed problems form #46

### CODE REVIEW

File: test/fixtures/mson-mixin-nonexistent.apib [hunk 1]
```diff
@@ -9,4 +9,4 @@
 
 ## User (object)
 - id
-- Include Address
+- Include XXX
```
Comment (pksunkara) [inline test/fixtures/mson-mixin-nonexistent.apib]: @klokane Snowcrash will give error when parsing this test case. It wouldn't print the AST.

Comment (klokane) [inline test/fixtures/mson-mixin-nonexistent.apib]: Hmm, with older version of snowcrash it was working (I forgot update sobmodules)

In future it should work again beacuse Mixins should allow reference remote blueprints. All checking in about referencing probably will be moved into drafter

Comment (pksunkara) [inline test/fixtures/mson-mixin-nonexistent.apib]: > with older version of snowcrash it was working

Yes, but I added this check during the bug-fixes.

> In future it should work again

Yes, but that's in the future.

Comment (klokane) [inline test/fixtures/mson-mixin-nonexistent.apib]: Hmm interesting
this: 
```
# <API name>
<API description>

# Data Structures

## User (object)
- id
- Include XXX 
```

Will not parse `Include XXX`

While this:
```
# <API name>
<API description>

# Data Structures

## User (object)

comment 

## Properties
- id
- Include XXX
```
does. 

(both cases returns error)

Comment (pksunkara) [inline test/fixtures/mson-mixin-nonexistent.apib]: When snowcrash returns an error, the AST it gives should be neglected. The AST should not even be tested when the parser has an error. This is why, the above 2 samples have different AST, because we are returning an error as soon as we see one.

Comment (klokane) [inline test/fixtures/mson-mixin-nonexistent.apib]: All tests in `test/test-Refract.cc` calling `HandleFixtureTest()` with second parameter `false` returns error while parsing.
It is part of checking edge condition
